### DISCUSSION TITLE
支付宝续订专业版遇到问题（2024年10月2日支付）

### DISCUSSION THREAD

**Discussion by MiuraAnjin** [getlantern/lantern-forum-cn#922] [Q&A]
订单号：2024100215005527733 [已通过支付宝支付，服务到期日没有延长。
支付后，网页提示： ip [111.192.16.193] 不在白名单。
10月2日已提工单：#77582418。请帮忙处理，谢谢。
![到期日](https://github.com/user-attachments/assets/246eb7a3-33c6-4d24-ade2-f7846a80b4b9)
![汇款截图](https://github.com/user-attachments/assets/c2f475e8-39e3-4344-9676-c133a493c8e5)

**Comment by MaxMKMax** [comment]
@MiuraAnjin 您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.

**Comment by MaxMKMax** [comment]
@MiuraAnjin 您好, 请您也提供我账号邮箱.

**Reply by MiuraAnjin** [reply]
wuxy21ce@sina.com

**Comment by MiuraAnjin** [comment]
您好，请问有新进展么？今天看到期日还是没有延长

**Reply by MaxMKMax** [reply]
@MiuraAnjin 来了. 我再帮您追问看看.

**Comment by MaxMKMax** [comment]
@MiuraAnjin 在等待他们处理呢. 上周已经帮您提交了.

**Reply by MiuraAnjin** [reply]
好的，感谢

**Comment by MiuraAnjin** [comment]
已经看到，到期日延长了2年，感谢！还希望能有支付时提示的，加赠3个月。
![到期日b](https://github.com/user-attachments/assets/b57a998b-4f7c-4c82-bb5a-d5342bb582f1)
